src/main/java/org/parabot/core/ui/components/VerboseLoader.java

```diff
@@ -21,7 +21,7 @@
 /**
  * An informative JPanel which tells the user what bot is doing
  *
- * @author Everel, EmmaStone
```

Comment:
Could you remove your changes of this file and take the changes from EmmaStone.

Simply copy pasta [this](https://github.com/Parabot/Parabot/blob/master/src/main/java/org/parabot/core/ui/components/VerboseLoader.java) into your file and push!